src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -322,6 +332,7 @@ public function __construct(array $options, HttpClientInterface $client = null)
 
         $this->options = $options;
         $this->client = $client ?: HttpClientFactory::create();
+        $this->messageFactory = $messageFactory ?: MessageFactoryDiscovery::find();
```

Comment:
... We need to require discovery. Or, make our own Httplug factory with the logic from CoreGatewayFactory